arches/app/models/resource.py

```diff
@@ -500,6 +500,26 @@ def get_documents_to_index(
         document["displayname"] = []
         document["displaydescription"] = []
         document["map_popup"] = []
+        try:
+            document["date_created"] = (
+                models.EditLog.objects.filter(
+                    resourceinstanceid=str(self.resourceinstanceid)
+                )
+                .earliest("timestamp")
+                .timestamp
+            )
+        except:
+            document["date_created"] = None
+        try:
+            document["date_last_edited"] = (
+                models.EditLog.objects.filter(
+                    resourceinstanceid=str(self.resourceinstanceid)
+                )
+                .latest("timestamp")
+                .timestamp
+            )
+        except:
+            document["date_last_edited"] = None
```

Comment:
I'm not sure how others feel, but I would not object to the 'lasteditedtime' also being added to the resource instance model. Otherwise this query is going to slow down indexing and is vulnerable to admins deleting their edit history. I think updating the last edited timestamp on the resource could easily be added to the `resource.save_edit` method.